Gemfile

```diff
@@ -35,6 +36,7 @@ gem 'cloudinary', '~> 1.16.0'
 gem 'bootsnap', '>= 1.4.2', require: false
 
 gem 'devise'
+gem 'cloudinary', '~> 1.16.0'
```

Comment:
why do we have cloudinary gem both in line 39 and in line 25?

Reply:
not sure, I put one. maybe the other one was preinstalled already